.yarn/versions/00a383da.yml

```diff
@@ -0,0 +1,25 @@
+releases:
+  "@yarnpkg/builder": major
+  "@yarnpkg/cli": major
```

Comment:
I don't see why the cli needs a release 🤔

Reply:
I bumped it because updating ESBuild might cause the generated bundle to be different :thinking:

Reply:
Oh yeah, due to esbuild not always including `__esModule` anymore it might break plugins that depends on it existing 🤔